frontend/src/app/signUp/IdentityVerification/PersonalDetailsForm.tsx

```diff
@@ -238,7 +238,7 @@ const PersonalDetailsForm = ({
               Identity verification helps protect organizations working with
               personal health information.
             </p>
-            <h3>{getPersonFullName()}</h3>
+            <h2 className="h3-override">{getPersonFullName()}</h2>
```

Comment:
nit: could we name this class to be something more semantically descriptive?

Reply:
I'd love a suggestion, I've been unable to understand SR's naming conventions for css 😁